rel/config.exs

```diff
@@ -0,0 +1,60 @@
+# Import all plugins from `rel/plugins`
+# They can then be used by adding `plugin MyPlugin` to
+# either an environment, or release definition, where
+# `MyPlugin` is the name of the plugin module.
+Path.join(["rel", "plugins", "*.exs"])
+|> Path.wildcard()
+|> Enum.map(&Code.eval_file(&1))
+
+use Mix.Releases.Config,
+    # This sets the default release built by `mix release`
+    default_release: :default,
+    # This sets the default environment used by `mix release`
+    default_environment: Mix.env()
+
+# For a full list of config options for both releases
+# and environments, visit https://hexdocs.pm/distillery/configuration.html
+
+
+# You may define one or more environments in this file,
+# an environment's settings will override those of a release
+# when building in that environment, this combination of release
+# and environment configuration is called a profile
+
+environment :dev do
+  # If you are running Phoenix, you should make sure that
+  # server: true is set and the code reloader is disabled,
+  # even in dev mode.
+  # It is recommended that you build with MIX_ENV=prod and pass
+  # the --env flag to Distillery explicitly if you want to use
+  # dev mode.
+  set dev_mode: true
+  set include_erts: false
+  set cookie: :")zw(v[=pk_@t8^[.lu>iXGjgaT;:?FbkFwSSJc]kTMT[X&~zuO*r]k;k4*zj1>.&"
+end
+
+environment :prod do
+  set include_erts: true
+  set include_src: false
+  set cookie: :"3b{RFjsEIeJ$}V/1Js$g/GvbH]?pWaig~lxpQgi>af1{N1P~:ump),5o~Jw:,LS="
```

Comment:
This should be secret token, right?

Reply:
Can and should be customized if needed. In docker images since distribution is not exposed it's a non issue.

Reply:
Also it can be set after building image too, by editing config. We'll setup cuttlefish sometime for a easier config format.